en/performance/practical-search-performance-guide.md

```diff
@@ -613,7 +613,7 @@ transferred during the summary fill phase.
 
 <pre>
 document-summary track_id {
-    summary track_id type string { 
+    summary track_id {
```

Comment:
I suggest removing the `source` specification as well to make the example as simple as possible. Applies to the rest of the sd file.